tensorflow_addons/image/__init__.py

```diff
@@ -17,6 +17,8 @@
 from __future__ import division
 from __future__ import print_function
 
+from tensorflow_addons.image.dense_image_warp import dense_image_warp
+from tensorflow_addons.image.dense_image_warp import _interpolate_bilinear
```

Comment:
If we're going to expose `interpolate_bilinear` can we remove the leading `_` so its not thought of as a private method.